AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -56,6 +56,8 @@ aksaelektrik.com.tr,ergo.de,fosforludusunceler.com,kirtkirtla.com,sonaktuel.com,
 !
 n-tv.de###gujems-cookie-banner
 kodak.com##.c_cookie_notice
+iobroker.net##.jss120
+iobroker.net##.jss110
```

Comment:
classes are different in safari and other browsers

Reply:
yes, mobile and desktop